contributor-guide/modules/ROOT/pages/requirements/organization-requirements.adoc

```diff
@@ -7,20 +7,22 @@ The quality of the Boost libraries is not just about the APIs and code design. B
 |*config*	|Files used for build-time configuration checks. This directory may contain source files and build system scripts to be used when building the library, tests or examples to check if the target system satisfies certain conditions. For example, a check may test if the compiler implements a certain feature, or if the target system supports a certain API.	|Optional.
 |*doc*	|Sources to build with and built documentation for the library. If the library needs to build documentation from non-HTML files this location must be build-able with Boost Build.	|Required for all libraries.
 |*doc/html*	|Documentation (HTML) files.	|Required for all libraries with pregenerated documentation. And generated documentation must be generated here.
 |*example*	|Sample program files.	|Required if library has sample files. Which is highly recommended.
-|*index.html*	|Redirection to HTML documentation. See "Redirection" for a template for this file.	|Required for all libraries.
+|*index.html*	|Redirection to HTML documentation. Refer to xref:design-guide/design-guide.adoc#redirection[Design Guide/Redirection] for a template for this file.	|Required for all libraries.
 |*include/boost/library*	|Header files for the library.	|Required for all libraries.
 |*meta*	|Meta-data about the library.	|Required for all libraries.
-|*meta/libraries.json*	|A JSON file containing information about the library used to generate website and documentation for the Boost C++ Libraries collection.	|Required for all libraries.
+|*meta/libraries.json*	|A JSON file containing information about the library, which is used to generate website and documentation for the Boost Libraries collection. Refer to xref:requirements/library-metadata.adoc[].	|Required for all libraries.
 |*meta/explicit-failures-markup.xml*	|XML file describing expected test failures, used to generate the test report.	|Optional
 |*src*	|Source files which must be compiled to build the library.	|Required if the library has source files to build.
 |*test*	|Regression or other test programs or scripts. This is the only location considered for automated testing. If you have additional locations that need to be part of automated testing it is required that this location refer to the additional test locations.	|Required for all libraries.
+|*test/cmake_install_test* | The sub-folder is named for the type of tests (`install` is just one example), and these sub-folders contain tests for xref:testing/continuous-integration.adoc[]. The documentation for the contents of this folder is a work in progress: https://github.com/boostorg/cmake/blob/develop/developer.md#ci-quick-testing[CI Quick Testing]. Consider referring to the `cmake_install_test` folder in https://github.com/vinniefalco/mp11/tree/52fd7817e85e2d208d364f2ff4bde74594308945/test[mp11/test], or https://github.com/boostorg/json/tree/66d925116740c1063eab814be7821fc7611ac480/test[json/test] as examples. | Not required, though most new libraries include these tests.
```

Comment:
Please ensure this matches the layout in https://github.com/boostorg/boost-ci

I know it doesn't match it in this case but have a look at other cases too. And refer to https://github.com/boostorg/boost-ci in this section it doesn't already.

Reply:
@alandefreitas Reworded, added links.